sources/packages/api/src/route-controllers/institution-locations/institution-location.controller.service.ts

```diff
@@ -70,4 +75,34 @@ export class InstitutionLocationControllerService {
       ? DesignationStatus.Designated
       : DesignationStatus.NotDesignated;
   }
+
+  /**
+   * Get the Institution Location details.
+   * @param institutionId
+   * @param locationId
+   * @returns location details as InstitutionLocationFormAPIOutDTO.
```

Comment:
As discussed among devs, we should avoid using the object type name in returns comment.